_announcements/01_apple_common_removal.md

```diff
@@ -0,0 +1,319 @@
+
+- [How Does this Work?](#how-does-this-work)
+- [What Will Be Impacted?](#what-will-be-impacted)
+- [What Should I Do?](#what-should-i-do)
+- [Frequently Asked Questions](#frequently-asked-questions)
+- [Additional Resources](#additional-resources)
+
+## How Does This Work?
+
+Apple currently distributes the Federal Common Policy CA (FCPCA/COMMON) through its pre-installed Operating System Trust Stores for iOS, macOS, and tvOS. 
+
+Three root CA certificate _types_ reside in Apple's Trust Stores:
+
+- _Trusted Certificates_ &mdash; Trusted certificates that establish a chain of trust.
+- _Always Ask_ &mdash; Untrusted certificates that are not blocked. If a resource (e.g., website or signed email) chains to one of these certificates, the Apple Operating System will ask you to choose whether or not to trust it.
+- _Blocked_ &mdash; Potentially compromised certificates that will never be trusted.
+
+These certificate types are stored within Apple _Keychains_:
+
+- _Login Keychain_ &mdash; Certificates associated with a user account logged into a device.
+- _System Keychain_ &mdash; Certificates associated with all user accounts on a device (similar to the Microsoft Windows' _Local Machine_ certificate store). 
+- _System Roots Keychain_ &mdash; Includes Apple's _pre-installed_, trusted root CA certificates. COMMON will be removed from this Keychain.
+ 
+## What Will Be Impacted?
+
+These Apple Operating System versions will be impacted:
+
+|**macOS**|**iOS**|**tvOS**|
+| :-------- |:-------- |:-------- |
+| High Sierra (10.13) |	 iOS11 | tvOS11 |
+| Sierra (10.12) | iOS10 | tvOS10 |
+| El Capitan (10.11) | iOS9 | |
+| Yosemite (10.10) | iOS8 | |
+| Mavericks (10.9) | iOS7 | |
+
+Government users will receive errors on government-furnished equipment if any of these are true:
+
+1. Logging into a government network with a PIV credential 
+2. Authenticating to a government Virtual Private Network (VPN) endpoint with a PIV credential
+3. Authenticating to an internet-facing, government collaboration portal with a PIV credential 
+4. Browsing with Safari, Chrome, or Edge/IE to a government **intranet** website that uses a Federal PKI CA-issued TLS/SSL certificate
+5. Opening an Apple Mail or Microsoft Outlook email that was digitally signed using a Federal PKI CA-issued certificate
+6. Opening a Microsoft Office document that was digitally signed with a Federal PKI CA-issued certificate
+
+This change will also impact Federal Government partners that rely on COMMON&mdash;for example, a Department of Defense employee sending a digitally signed email to a business partner. 
+
+You can mitigate the risk to government missions, intranets, applications, and government-furnished equipment.
+
+{% include alert-info.html content="If you are unsure whether your applications will be affected, email us at fpki@gsa.gov." %} 
+
+## What Should I Do?
+
+These procedures are intended for Enterprise Administrators and/or Network Engineers. 
+
```

Comment:
Can we add the sub-links to the sub-sections?  Download and Install are two separate sets of steps and options.   

I also can't tell if it's broken down by OS / device type, or actions. 

It's a bit confusing when sub-sub-sections are all "Option 1" "Option 2" etc.

Reply:
I had tried this but ran into issues (links did not appear). I'll try again!